content/news/2024-11-04-biohackathon/index.md

```diff
@@ -0,0 +1,70 @@
+
+The main projects that interacted with Galaxy were:
+
+# [Project 20](https://github.com/elixir-europe/biohackathon-projects-2024/blob/main/11.md)
+
+Building on the achievements of BioHackathon Europe 2023, we successfully enhanced the Galaxy ecosystem by extending the capabilities of the existing Galaxy tool list table. A major milestone was addition of workflows and training leading to the the creation and of the Galaxy Communities Dock (Galaxy Codex).
+This new resource also integrates the Galaxy Labs and allows to automatically add community resources.
+
+This project was led by Bérénice Batut and Wendi Bacon. Paul Zierep helped with updates to the CI and minor code improvements.
+
+The updates are available via the [Galaxy CoDex](https://github.com/galaxyproject/galaxy_codex) GitHub repository.
+
+# [Project 21](https://github.com/elixir-europe/biohackathon-projects-2024/blob/main/21.md)
+
+This project worked on a vision of a “federated microbiome analysis service”.
+Paul Zierep participated in the discussion involving Galaxy and Alireza Heidari supported front-end related coding.
+
+A preprint is available (Enhancing multi-omic analyses through a federated
+microbiome analysis service)[http://preview.biohackrxiv.org/papers/154602ea-ae08-42c0-bb65-0b737d282207/paper.pdf]
+
+# [Project 22](https://github.com/elixir-europe/biohackathon-projects-2024/blob/main/22.md)
+
+Project Objectives
+The objective of the project can be divided into two milestones:
+
+* Development of a workflow that connects EGA, either central or any federated node, and Galaxy through Crypt4GH protocols.
+* Galaxy's secure processing protocol: Sensitive datasets are kept encrypted throughout, with sensitive derivative results labelled as sensitive.
+
+# [Project 26](https://github.com/elixir-europe/biohackathon-projects-2024/blob/main/26.md)
+We focused on making computational data analysis more environmentally sustainable through improvements to Galaxy. By leveraging Galaxy's control over resource usage, we aimed to reduce the environmental impact of workflows via two key initiatives: job caching and environmentally-friendly job scheduling.
+### Smarter Resource Use with Job Caching
+
+Galaxy already tracks the provenance of every tool execution, and we used this information to avoid redundant computations. Job caching enables Galaxy to reuse existing outputs when input parameters and conditions match, saving time and resources. During the Biohackathon, we worked to extend this feature to handle more scenarios. For example, we are refining how dataset metadata is matched to determine whether a job can be reused.
+
+Additionally, we explored ways to enable shared caching among users who opt in. This will be especially useful in collaborative settings, such as training sessions, where large-scale analyses can be performed without unnecessary computational overhead.
```

Comment:
```suggestion
Additionally, we explored ways to enable shared caching among users who opt-in. This will be especially useful in collaborative settings, such as training sessions, where large-scale analyses can be performed without unnecessary computational overhead.
```